script/dist-info.js

```diff
@@ -78,6 +79,12 @@ function getWindowsIdentifierName () {
   return 'GitHubDesktop'
 }
 
+function getBundleSizes () {
+  const rendererStats = fs.statSync(path.join(projectRoot, 'out', 'renderer.js'))
```

Comment:
Really not a huge deal, but `path.join(projectRoot, 'out')` can be put in a constant .